sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -180,9 +191,17 @@ export class Receiver {
     cancellationToken?: Aborter
   ): Promise<ReceivedEventData[]> {
     this._throwIfReceiverOrConnectionClosed();
+    this._throwIfAlreadyReceiving();
+    if (this.getCheckpoint()) {
```

Comment:
`this.getCheckpoint()` is getting called thrice inside `receiveBatch()`, lets use a variable instead?